### ISSUE TITLE
npx @preset/cli apply hybridly/preset issues with Laravel 11??

### ISSUE THREAD

**Issue by MotionPhix** [hybridly/hybridly#126]
### Describe the bug

I am new to Hybridly and I wanted to setup in a Laravel 11 new project. When I run *npx @preset/cli apply hybridly/preset* command, I am getting

```
✘ [ERROR] Expected "]" but found "web"

    ../../../Users/User/AppData/Local/Temp/presets/preset/preset.ts:91:34:
      91 │ ... '$middleware->appendToGroup('web', [\\App\\Http\\Middleware\\...
         │                                  ~~~
         ╵                                  ]


 ERROR  Build failed with 1 error:
../../../Users/User/AppData/Local/Temp/presets/preset/preset.ts:91:34: ERROR: Expected "]" but found "web"
```

Should we say I am doing something wrong or there's no support for Laravel 11 currently?

On the other hand, I need to ask if running the above command is that all I need to get hybridly to work, because in the documentation I saw another section that said server side setup.

### Reproduction

github.com/MotionPhix

### Steps to reproduce

Create a new Laravel App and then run *npx @preset/cli apply hybridly/preset*

### System information

```bash
Windows 10
```

### Used package manager

npm

### Validations

- [X] Read the [docs](https://hybridly.dev).
- [X] Check that there isn't [already an issue](https://github.com/hybridly/hybridly/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Hybridly issue and not an issue related to something else (Vite, Vue...). For example, if it's a Vue SFC related bug, it should likely be reported to [vuejs/core](https://github.com/vuejs/core) instead.
- [X] Check that this is a concrete bug. For Q&A, join our [Discord server](https://discord.gg/uZ8eC7kRFV).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by innocenzi** [comment]
It was a typo in our preset, thanks for reporting! Should be working now.

**Comment by MotionPhix** [comment]
Thank you for the quick response. Truly appreciate. While the above issue has been definitely resolved, I am yet facing another error. The error says:

```
▲ [WARNING] Cannot find base config file "./.hybridly/tsconfig.json" [tsconfig.json]

    tsconfig.json:2:12:
      2 │   "extends": "./.hybridly/tsconfig.json"
        ╵              ~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

How would I get rid of this?

**Comment by innocenzi** [comment]
This error is harmless, it's due to Hybridly generating `tsconfig.json` after it's ran (for the first time). Everything should still work fine 👍

**Comment by cgslivre** [comment]
Congratulations on your work @innocenzi !

I tried to test it by following the tutorial described at:
https://hybridly.dev/guide/installation.html#preset

I get the following package compatibility error:
```bash
bunx @preset/cli apply hybridly/preset
```
```bash
  ERROR  Failed applying laravel:hybridly.

  ✓  Executed action: install Pest (preset: laravel:pest)
    ✓  Executed action: register the composer plugin
    ✓  Executed action: remove phpunit
    ✓  Executed action: install dependencies (php)
    ✓  Executed action: setup Pest
  ✓  Executed action: delete unused project files
  ✓  Executed action: update .gitignore
  ✓  Executed action: add middleware
  ✓  Executed action: extract templates
  ✓  Executed action: remove unused dependencies
  ✓  Executed action: add npm dependencies (node)
  ✓  Executed action: configure PostCSS
  ×  Failed action: add php dependencies (php)
     ↪  Command failed with exit code 2: composer require --no-interaction hybridly/laravel spatie/laravel-data
      ./composer.json has been updated
      Running composer update hybridly/laravel spatie/laravel-data
      Loading composer repositories with package information
      Updating dependencies
      Your requirements could not be resolved to an installable set of packages.

        Problem 1
          - hybridly/laravel[v0.1.0, ..., v0.6.0] require illuminate/contracts ^9.0|^10.0 -> found illuminate/contracts[v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.22] but these were not loaded, likely because it conflicts with another require.
          - hybridly/laravel v0.6.1 requires illuminate/contracts ^10.0 -> found illuminate/contracts[v10.0.0, ..., v10.48.22] but these were not loaded, likely because it conflicts with another require.
          - hybridly/laravel[v0.7.0, ..., v0.7.14] require laravel/prompts ^0.1 -> found laravel/prompts[v0.1.0, ..., v0.1.25] but the package is fixed to v0.3.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument
for the update command.
          - Root composer.json requires hybridly/laravel * -> satisfiable by hybridly/laravel[v0.1.0, ..., v0.7.14].

      Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
      You can also try re-running composer require with an explicit version constraint, e.g. "composer require hybridly/laravel:*" to figure out if any version is installable, or "composer require hybridly/laravel:^2.1" if you know which you need.

      Installation failed, reverting ./composer.json and ./composer.lock to their original content.
  ✓  Executed action: add php dev dependencies (php)
  ✓  Executed action: publish typescript transformer config
  ✓  Executed action: update typescript transformer config
  ✓  Executed action: update welcome route
  ✓  Executed action: apply strict mode
  ✓  Executed action: setup ide helper (2 actions)

 Presets  1 applied | 1 failed (2)
 Actions  21 executed | 1 failed (22)
Duration  58.51s


  ERR_PRESET_FAILED  Some of the actions of the preset failed.
```

I tried with: yarn, npm and bun, without success!

php: 8.2
fresh laravel 11 via:
composer create-project laravel/laravel hybridly-app
